tools/slo-generator/tests/unit/test_compute.py

```diff
@@ -12,203 +12,117 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import yaml
-import os
 import unittest
-import mock
-import pprint
-import string
-import time
-from google.cloud.monitoring_v3.proto import metric_service_pb2
-from slo_generator.exporters.bigquery import BigQueryError
-from slo_generator.compute import (compute, export, make_reports,
-                                   make_measurement)
-
-cwd = os.path.dirname(os.path.abspath(__file__))
-
-FIXTURES_CONFIG = {
-    'PUBSUB_PROJECT_ID': 'fake',
-    'PUBSUB_TOPIC_NAME': 'fake',
-    'STACKDRIVER_HOST_PROJECT': 'fake',
-    'STACKDRIVER_METRIC_PROJECT': 'fake',
-    'BIGQUERY_HOST_PROJECT': 'fake',
-    'BIGQUERY_TABLE_ID': 'fake',
-    'BIGQUERY_DATASET_ID': 'fake',
-    'BIGQUERY_TABLE_NAME': 'fake'
-}
-
-
-class MultiCallableStub(object):
-    """Stub for the grpc.UnaryUnaryMultiCallable interface."""
-
-    def __init__(self, method, channel_stub):
-        self.method = method
-        self.channel_stub = channel_stub
-
-    def __call__(self, request, timeout=None, metadata=None, credentials=None):
-        self.channel_stub.requests.append((self.method, request))
-
-        response = None
-        if self.channel_stub.responses:
-            response = self.channel_stub.responses.pop()
-
-        if isinstance(response, Exception):
-            raise response
-
-        if response:
-            return response
-
-
-class ChannelStub(object):
-    """Stub for the grpc.Channel interface."""
+import warnings
 
-    def __init__(self, responses=[]):
-        self.responses = responses
-        self.requests = []
-
-    def unary_unary(self,
-                    method,
-                    request_serializer=None,
-                    response_deserializer=None):
-        return MultiCallableStub(method, self)
-
-
-def dummy_slo_function(timestamp, window, **kwargs):
-    return (300, 2)
-
-
-class DummySLOBackend(object):
-
-    def __init__(self, **kwargs):
-        pass
-
-    def dummy_slo_function(self, timestamp, window, **kwargs):
-        return (300, 2)
+from elasticsearch import Elasticsearch
+from google.auth._default import _CLOUD_SDK_CREDENTIALS_WARNING
+from mock import patch, MagicMock
+from prometheus_http_client import Prometheus
+from slo_generator.exporters.bigquery import BigQueryError
+from slo_generator.compute import compute, export
+from test_stubs import (CTX, load_fixture, load_sample, load_slo_samples,
+                        mock_sd, mock_prom, mock_es, mock_ssm_client)
+
+warnings.filterwarnings("ignore", message=_CLOUD_SDK_CREDENTIALS_WARNING)
+
+ERROR_BUDGET_POLICY = load_sample('error_budget_policy.yaml', **CTX)
```

Comment:
File was deleted?

Reply:
`load_sample` now loads all samples from the `samples/` directory, so this file does exist (cf. `samples/error_budget_policy.yaml`)